src/library/Uno.Material/Styles/Controls/v2/ComboBox.xaml

```diff
@@ -463,8 +463,6 @@
 					<Grid x:Name="RootGrid"
 						  HorizontalAlignment="{TemplateBinding HorizontalAlignment}"
 						  Background="{TemplateBinding Background}"
-						  BorderBrush="{TemplateBinding BorderBrush}"
-						  BorderThickness="{TemplateBinding BorderThickness}"
 						  CornerRadius="{ThemeResource ComboBoxCornerRadius}">
 						<VisualStateManager.VisualStateGroups>
```

Comment:
background and cornerradius here should both be moved onto `ComboBoxContent`

Reply:
the HorizontalAlignment can simply be removed, it is unnecessary
since the combo's HA already confines it
x:Name doesnt seem to be used neither, could remove too
all together, you should be left with a `<Grid>` with no attribute